### DISCUSSION TITLE
Force stitching

### DISCUSSION THREAD

**Discussion by HugoDeCozar** [OpenStitching/stitching#151] [Q&A] [answered]
Hello, its me again.

Is it there a way to force the stitching if we know the order of the images? These two photos clearly are part of the same panoramic, but it is impossible to stitch them. I have tried every detector and match conf.

Thanks!

<img width="1325" alt="left" src="https://github.com/OpenStitching/stitching/assets/145971234/76dc80c0-4437-4083-86cb-d56f9fec5138">

<img width="1319" alt="right" src="https://github.com/OpenStitching/stitching/assets/145971234/2cdf1fcc-f5df-4c57-9191-d1d6e630fa09">

**Comment by lukasalexanderweber** [comment]
You might need to reconsider your setup. The images have very little overlap with matching features especially in the background. I also tried a bit but havn't found a working parameter set. 

![01_features_img1](https://github.com/OpenStitching/stitching/assets/32765578/a1d8c7c7-6567-4e42-ac81-ec7ee08761f7)
![01_features_img2](https://github.com/OpenStitching/stitching/assets/32765578/4742e8d8-297c-4d0b-947c-a69fc281055f)
![02_matches_img1_to_img2](https://github.com/OpenStitching/stitching/assets/32765578/ff8101b1-aa3f-4bcf-9f14-d40a4e8e75b9)

**Reply by HugoDeCozar** [reply]
I've tried marking the overlapping part of both images, placing shapes in the exact same spots so the algorithm picks up more matching features, but it didn't work.

**Reply by lukasalexanderweber** [reply, marked as answer]
The library still missing the feature to manually select matching points, this example would be a good fit for it
